plugins/hybrid_ae_pkg/hybrid_ae_templates/hybrid_ae_template.html

```diff
@@ -0,0 +1,51 @@
+{% import 'forms.html' as forms %}
+
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='forms.css')}}" media="screen" />
+
+    <style type="text/css">
+        @import url("{{url_for('static', filename='forms.css')}}");
+    </style>
```

Comment:
The style tag can be removed as the style sheet is already loaded with the link in line 6. Using a css import in a style tag was neccessary with the old style micro frontends as they were only html fragments.